modules/react/loading-sparkles/lib/sparkleStage0Icon.tsx

```diff
@@ -0,0 +1,10 @@
+import {CanvasSystemIcon, CanvasIconTypes} from '@workday/design-assets-types';
+
+export const sparkleStage0Icon: CanvasSystemIcon = {
+  name: 'sparkle-stage-0',
+  type: CanvasIconTypes.System,
+  svg: `<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M6 0C6.15942 0 6.34301 0.0967055 6.38274 0.308112C6.64764 1.72077 7.21903 2.91761 8.0963 3.82655C8.97292 4.73482 10.1632 5.36427 11.6812 5.62996C12.1068 5.70416 12.1058 6.29577 11.6812 6.37004C10.1632 6.63573 8.97292 7.26518 8.0963 8.17345C7.21903 9.08239 6.64765 10.2792 6.38276 11.6918C6.34303 11.9032 6.15942 12 6 12C5.84058 12 5.65699 11.9033 5.61726 11.6919C5.35236 10.2792 4.78097 9.08245 3.9037 8.17353C3.02708 7.26528 1.8368 6.63583 0.31883 6.37004C-0.106757 6.29584 -0.105799 5.70423 0.31884 5.62996C1.8368 5.36427 3.02708 4.73482 3.9037 3.82655C4.78097 2.91761 5.35235 1.72083 5.61724 0.308177C5.65697 0.0967702 5.84058 0 6 0Z" class="wd-sparkle-fill"/>
+  </svg>`,
+  filename: 'wd-icon-sparkle-stage-0.svg',
+};
```

Comment:
Since all other icons in this repo pull from `@workday/canvas-system-icons-web`, I'm not sure if there's a better pattern for these assets here (or if they should be moved into that package). Each SVG represents a slightly different shape of the main sparkle icon.

Reply:
If I'm not mistaken we do plan to add these to `@workday/canvas-system-icons-web` but they will not be available till middle of Jan